src/createReducer.ts

```diff
@@ -25,11 +38,11 @@ const createReducer = (...middlewares) => {
       [reducer]
     );
 
-    const dispatchRef = useRef(
+    const dispatchRef: MutableRefObject<Dispatch<Action>> = useRef(
       composedMiddleware(
         {
           getState: () => ref.current,
-          dispatch: (...args) => dispatchRef.current(...args),
+          dispatch: (...args: [Action]) => dispatchRef.current(...args),
```

Comment:
If it is just a single argument, do we need spreading here?

```suggestion
          dispatch: (action: Action) => dispatchRef.current(action),
```